UPGRADE-6.2.md

```diff
@@ -86,6 +86,10 @@ Security
  * Add maximum username length enforcement of 4096 characters in `UserBadge` to
    prevent [session storage flooding](https://symfony.com/blog/cve-2016-4423-large-username-storage-in-session)
  * Deprecate the `Symfony\Component\Security\Core\Security` class and service, use `Symfony\Bundle\SecurityBundle\Security\Security` instead
+ * Deprecate the `Symfony\Bundle\SecurityBundle\Security\Security::ACCESS_DENIED_ERROR` property, use `Symfony\Component\Security\Http\SecurityRequestAttributes::ACCESS_DENIED_ERROR` instead
```

Comment:
This looks wrong :)
- `Symfony\Bundle\SecurityBundle\Security\Security` doesn't exist (`Symfony\Bundle\SecurityBundle\Security` does)
- What is actually deprecated is: `Symfony\Component\Security\Core\Security::*` constants in favor of `Symfony\Bundle\SecurityBundle\Security::*` equivalents (they will stay)